apache/superset
# feat(legacy-plugin-chart-country-map): added support for Legend options

### SUMMARY
Country map seems to be not very clear when you have to see and check why specific color has been chosen for an area of the map. I think legend option is mandatory to improve readability of charts. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before was not possible to configure legend, now you can use specific parameters to obtain it:
<img width="1360" alt="image" src="https://user-images.githubusercontent.com/2301162/206148326-194a838d-91a5-4a92-9a4e-c1bec0223098.png">

I've added few new config props:
- **LEGEND**: allow to set position of the legend (_None, Top, TopLeft, TopRight, Bottom, BottomLeft, BottomRight_)
- **ORIENTATION**: vertical or horizontal (like a color gradient)
- **FONT SIZE**: used to render each step of the legend.

### TESTING INSTRUCTIONS

Run the application and create a new chart of type _Country Map_, you will see new parameters that can be used to customize visualization of the chart.

### ADDITIONAL INFORMATION
